test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/SubWindowTest.cs

```diff
@@ -1,102 +1,142 @@
 
 namespace Tizen.NUI.Samples
 {
     using log = Tizen.Log;
     public class SubWindowTest : IExample
     {
-        string tag = "NUITEST";
-        Window mainWin;
-        Window subWin1;
-        Window subWin2;
-        Window subWin3;
-        Timer tm;
-        bool belowParent;
-        void Initialize()
+        private const string tag = "NUITEST";
+        private const string keyBack = "XF86Back";
+        private const string keyEsc = "Escape";
+        private const string key1 = "1";
+        private const string key2 = "2";
+        private const string key3 = "3";
+        private const string key4 = "4";
+        private const string key5 = "5";
+        private const string key6 = "6";
+        private const string key7 = "7";
+        private Window mainWin;
+        private Window subWinFirst;
+        private Window subWinSecond;
+        private Window subWinThird;
+        private Timer timer;
+        private bool belowParent;
+        private const int oneSecond = 1000;
+        private const int testSize = 100;
+        private const int testPosition = 100;
+        private const float testPointSize = 12.0f;
+
+        public void Activate()
+        {
+            Initialize();
+        }
+
+        public void Deactivate()
+        {
+        }
+
+        private void Initialize()
         {
             mainWin = NUIApplication.GetDefaultWindow();
             mainWin.KeyEvent += OnKeyEvent;
             mainWin.BackgroundColor = Color.Cyan;
-            mainWin.WindowSize = new Size2D(500, 500);
+            mainWin.WindowSize = new Size2D(5 * testSize, 5 * testSize);
             mainWin.TouchEvent += WinTouchEvent;
             belowParent = false;
 
-            TextLabel text = new TextLabel("Hello Tizen NUI World");
-            text.HorizontalAlignment = HorizontalAlignment.Center;
-            text.VerticalAlignment = VerticalAlignment.Center;
-            text.TextColor = Color.Blue;
-            text.PointSize = 12.0f;
-            text.HeightResizePolicy = ResizePolicyType.FillToParent;
-            text.WidthResizePolicy = ResizePolicyType.FillToParent;
+            TextLabel text = new TextLabel("Hello Tizen NUI World")
+            {
+                HorizontalAlignment = HorizontalAlignment.Center,
+                VerticalAlignment = VerticalAlignment.Center,
+                TextColor = Color.Blue,
+                PointSize = testPointSize,
+                HeightResizePolicy = ResizePolicyType.FillToParent,
+                WidthResizePolicy = ResizePolicyType.FillToParent
+            };
             mainWin.Add(text);
 
-            Animation animation = new Animation(2000);
-            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(180.0f)), PositionAxis.X), 0, 500);
-            animation.AnimateTo(text, "Orientation", new Rotation(new Radian(new Degree(0.0f)), PositionAxis.X), 500, 1000);
+            Animation animation = new Animation(2 * oneSecond);
```

Comment:
animation is meaningless variable name.
what about testRotationAnim?

Reply:
Thank for review. I fixed it.